packages/react-native-renderer/src/ReactNativeFiberInspector.js

```diff
@@ -117,13 +150,91 @@ if (__DEV__) {
       source,
     };
   };
+
+  getInspectorDataForViewAtPoint = function(
+    findNodeHandle,
+    inspectedView: ?React.ElementRef<ReactNativeHostComponent<mixed>>,
+    x: number,
+    y: number,
+    callback: (obj: Object) => mixed,
+  ): void {
+    let closestInstance = null;
+    let frame = {
+      left: 0,
+      top: 0,
+      width: 0,
+      height: 0,
+    };
+
+    if (inspectedView._internalInstanceHandle != null) {
+      // For Fabric we can look up the instance handle directly and measure it.
+      nativeFabricUIManager.findNodeAtPoint(
+        inspectedView._internalInstanceHandle.stateNode.node,
+        x,
+        y,
+        shadowNode => {
+          if (shadowNode == null) {
+            callback(getInspectorDataForInstance(closestInstance, frame));
+          }
+
+          closestInstance =
+            shadowNode.stateNode.canonical._internalInstanceHandle;
+          nativeFabricUIManager.measure(
+            shadowNode.stateNode.node,
+            (x, y, width, height, pageX, pageY) => {
+              frame = {
+                left: pageX,
+                top: pageY,
+                width,
+                height,
+              };
+
+              callback(getInspectorDataForInstance(closestInstance, frame));
+            },
+          );
+        },
+      );
+    } else if (inspectedView._internalFiberInstanceHandle != null) {
+      // For Paper we fall back to the old strategy using the React tag.
+      UIManager.findSubviewIn(
+        findNodeHandle(inspectedView),
+        [x, y],
+        (nativeViewTag, left, top, width, height) => {
+          frame = {
+            left,
+            top,
+            width,
+            height,
+          };
+          const inspectorData = getInspectorDataForInstance(
+            getClosestInstanceFromNode(nativeViewTag),
```

Comment:
This is the first line in `getInspectorDataForViewTag` (the rest of that function is the same as `getInspectorDataForInstance`), I do this here so that it's easier to delete the tag version later (after this change that function is only exported.